common/src/main/java/com/wynntils/models/lootrun/LootrunModel.java

```diff
@@ -70,18 +74,40 @@ public class LootrunModel extends Model {
     private LootrunFinishedEventBuilder.Completed lootrunCompletedBuilder;
     private LootrunFinishedEventBuilder.Failed lootrunFailedBuilder;
 
+    // Data that can live in memory, when joining a class we will parse these
     private LootrunningState lootrunningState = LootrunningState.NOT_RUNNING;
     private LootrunLocation currentLocation;
     private LootrunTaskType currentTaskType;
-    private Map<BeaconColor, Integer> currentLootrunBeacons = new HashMap<>();
-    private Beacon currentBeacon;
+
+    // Data to be persisted
+    @RegisterStorage
+    private Storage<Map<String, Map<BeaconColor, Integer>>> currentLootrunSelectedBeaconStorage =
```

Comment:
There is a lot of `current` prefixes in the lootrun variables. Maybe this can be implicitly understood?

Reply:
I guess yes.

Reply:
But then maybe take away lootrun as well?

Reply:
Yes, you can drop that as well.